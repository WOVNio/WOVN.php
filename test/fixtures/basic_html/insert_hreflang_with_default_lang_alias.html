<!DOCTYPE html><head></head><body><p>example</p></body></html>
