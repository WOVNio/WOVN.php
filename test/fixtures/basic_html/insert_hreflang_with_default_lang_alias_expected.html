<!DOCTYPE html><head><link rel="alternate" hreflang="en" href="http://my-site.com/english/"><link rel="alternate" hreflang="vi" href="http://my-site.com/vi/"><script src="//j.wovn.io/1" data-wovnio="key=123456&amp;backend=true&amp;currentLang=en&amp;defaultLang=en&amp;urlPattern=path&amp;langCodeAliases={&quot;en&quot;:&quot;english&quot;}&amp;langParamName=wovn" data-wovnio-info="version=WOVN.php_VERSION" async></script></head><body><p>example</p></body></html>
